packages/components/src/flyout/flyout-content/component.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Internal dependencies
+ */
+import { useFlyoutContext } from '../context';
+import { FlyoutContentView, CardView } from '../styles';
+import { contextConnect, useContextSystem } from '../../ui/context';
+
+/**
+ *
+ * @param {import('../../ui/context').PolymorphicComponentProps<import('../types').ContentProps, 'div', false>} props
```

Comment:
The refactoring done in https://github.com/WordPress/gutenberg/pull/32197/commits/3598b735319eaedc5b498e7d470a3b9f2d527243, (where we wrapped `ReakitPopover` directly into the `styled` function), caused a typescript error around the `as` prop on `< FlyoutContentView />`.

For this reason, `FlyoutContent` has been marked as non-polymorphic

Reply:
Out of curiosity, what's the context behind refactoring the `css` calls to `styled.div` and then to `styled( ReakitPopover )`? I remember we've had some conversations about that, but I can't remember exactly the reasoning behind it. cc @sarayourfriend

Reply:
Basically `@emotion/css` does not hook into the Emotion cache so it's not possible for us to use it's `css` function because we need to hook into the cache to support iframes.

`@emotion/styled` does hook into the CacheProvider so it is able to support iframes.

Reply:
Ah yeah! That makes sense.